src/gamestarfield.cpp

```diff
@@ -152,7 +152,24 @@ MOBase::VersionInfo GameStarfield::version() const
 
 QList<PluginSetting> GameStarfield::settings() const
 {
-  return QList<PluginSetting>();
+  return QList<PluginSetting>()
+         << PluginSetting("enable_plugin_management",
+                          tr("Turn on plugin management. As of Starfield 1.7.33 this "
+                             "REQUIRES SPECIAL WORKAROUNDS."),
```

Comment:
Want to mention the mod name directly or link it? It would be nice to have it somewhere so that users can find what the Special Workarounds are.

Reply:
It's a tooltip you only get with hovering over the option for a setting that's intentionally buried for people that hopefully already know what they're looking for.

I wasn't sure it felt right to mention a specific mod for that reason.

Reply:
I wouldn't be surprised if this became very popular when xEdit comes out or even before.
Maybe I can tell Nukem to add a screenshot to how to enable support in MO2 to his mod page when we release.